src/awkward/_v2/index.py

```diff
@@ -1,79 +1,84 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+from __future__ import absolute_import
+
```

Comment:
All the Python files start this way. (The "future import" is for Python 2.7 to have the same import semantics as Python 3.)